Jellyfin.Api/Helpers/TranscodingJobHelper.cs

```diff
@@ -607,6 +607,10 @@ private void DeleteHlsPartialStreamFiles(string outputFilePath)
             {
                 StartThrottler(state, transcodingJob);
             }
+            else if (transcodingJob.ExitCode != 0)
+            {
+                throw new Exception(string.Format(CultureInfo.InvariantCulture, "FFmpeg exited with code {0}", transcodingJob.ExitCode));
```

Comment:
I know we shouldn’t throw the base Exception, but I don’t know what would be better here

Reply:
Yeah, there is a `FfmpegException`, but I cannot reach it.

Reply:
Maybe we should move it to another project? (.Common for example)